### ISSUE TITLE
iOS Version support - automaticallyWaitsToMinimizeStalling

### ISSUE THREAD

**Issue by tspent** [feedfm/iOS-SDK#2]
Hey Guys,
Whats iOS version do you support?  We were using feedFm with no issues, but in your latest version
you are using:
- automaticallyWaitsToMinimizeStalling, which is iOS 10+. 

Our targets support iOS 8 +. Everyone is crashing!  

Can you please share with us in which version you start using it so we can revert, or check if the target is < 10 before you use the property.
Thanks

**Comment by eml** [comment]
Hi @tspent  -
  in v4.0.3 the library was updated to test for the presence of automaticallyWaitsToMinimizeStalling like so:

```
        if ([_player respondsToSelector:@selector(setAutomaticallyWaitsToMinimizeStalling:)]) {
            _player.automaticallyWaitsToMinimizeStalling = NO;
        }
```

so you shouldn't get a crash from that particular item - can you confirm you are using v4.0.3+? We do a similar test and conditional execution with MPMediaItemArtwork initWithBoundsSize:requestHandler:. 

In the 4.x versions we stopped testing iOS versions previous to 9.0 since we didn't know of any customers supporting pre-9.0, but DeployMate tells me that (aside from those two things above) we aren't using anything unavailable in 8.0.

**Comment by tspent** [comment]
Hi @eml,
Thanks for your fast response. We are using 4.0.0 as we support iOS 8 and your podsec states
 s.platform     = :ios, '9.0'. So we can't use your final version.

**Comment by eml** [comment]
Oops - that is a mistake on our end.

I just updated the podspec to indicate we support back to 8.0, and published this as version 4.0.7. Let me know if this doesn't solve things for you.
